exercises/scrabble-score/src/example.c

```diff
@@ -0,0 +1,50 @@
+#include "scrabble_score.h"
+#include <ctype.h>
+
+struct letter_score {
+   char letter;
+   unsigned int score;
+};
+
+static struct letter_score score_book[] = {
+   {'a', 1},
+   {'b', 3},
+   {'c', 3},
+   {'d', 2},
+   {'e', 1},
+   {'f', 4},
+   {'g', 2},
+   {'h', 4},
+   {'i', 1},
+   {'j', 8},
+   {'k', 5},
+   {'l', 1},
+   {'m', 3},
+   {'n', 1},
+   {'o', 1},
+   {'p', 3},
+   {'q', 10},
+   {'r', 1},
+   {'s', 1},
+   {'t', 1},
+   {'u', 1},
+   {'v', 4},
+   {'w', 4},
+   {'x', 8},
+   {'y', 4},
+   {'z', 10}
+};
+
+unsigned int score(const char *word)
+{
+   char c;
+   struct l;
+   unsigned int word_score = 0;
+
+   while ((c = tolower(*word++)) != '\0') {
+      struct letter_score l = score_book[c - 'a'];
```

Comment:
This is better, but now the `char` in `struct letter_score` is unused and unnecessary.